rtos/rtos.h

```diff
@@ -25,6 +25,9 @@
 #ifndef RTOS_H
 #define RTOS_H
 
+#include "mbed_rtx.h"
```

Comment:
it still provides the INITIAL_SP, I don't think there's generic way of extracting that of linker script and what's at address 0 may not be valid anymore.